{
	"document_title": "python310-rope-1.12.0-1.3 on GA media",
	"document_type": "SUSE Patch",
	"document_publisher": {
		"type": "Vendor",
		"contact_details": "security@suse.de",
		"issuing_authority": "SUSE Security Team"
	},
	"document_tracking": {
		"identification": {
			"id": "openSUSE-SU-2024:14161-1"
		},
		"status": "Final",
		"version": "1",
		"revision_history": {
			"revision": {
				"number": "1",
				"date": "2024-07-12T00:00:00Z",
				"description": "current"
			}
		},
		"initial_release_date": "2024-07-12T00:00:00Z",
		"current_release_date": "2024-07-12T00:00:00Z",
		"generator": {
			"engine": "cve-database/bin/generate-cvrf.pl",
			"date": "2017-02-24T01:00:00Z"
		}
	},
	"document_notes": [
		{
			"text": "python310-rope-1.12.0-1.3 on GA media",
			"title": "Topic",
			"type": "Summary"
		},
		{
			"text": "These are all security issues fixed in the python310-rope-1.12.0-1.3 package on the GA media of openSUSE Tumbleweed.",
			"title": "Details",
			"type": "General"
		},
		{
			"text": "The CVRF data is provided by SUSE under the Creative Commons License 4.0 with Attribution (CC-BY-4.0).",
			"title": "Terms of Use",
			"type": "Legal Disclaimer"
		},
		{
			"text": "openSUSE-Tumbleweed-2024-14161",
			"title": "Patchnames",
			"type": "Details"
		}
	],
	"documentdistribution": "Copyright SUSE LLC under the Creative Commons License 4.0 with Attribution (CC-BY-4.0)",
	"document_references": [
		{
			"type": "Self",
			"url": "https://www.suse.com/support/security/rating/",
			"description": "SUSE Security Ratings"
		},
		{
			"type": "Self",
			"url": "https://www.suse.com/security/cve/CVE-2014-3539/",
			"description": "SUSE CVE CVE-2014-3539 page"
		}
	],
	"product_tree": {
		"branch": [
			{
				"type": "Product Family",
				"name": "openSUSE Tumbleweed",
				"branch": [
					{
						"type": "Product Name",
						"name": "openSUSE Tumbleweed",
						"full_product_name": {
							"text": "openSUSE Tumbleweed",
							"product_id": "openSUSE Tumbleweed",
							"cpe": "cpe:/o:opensuse:tumbleweed"
						}
					}
				],
				"full_product_name": {}
			},
			{
				"type": "Product Version",
				"name": "python310-rope-1.12.0-1.3",
				"full_product_name": {
					"text": "python310-rope-1.12.0-1.3",
					"product_id": "python310-rope-1.12.0-1.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python311-rope-1.12.0-1.3",
				"full_product_name": {
					"text": "python311-rope-1.12.0-1.3",
					"product_id": "python311-rope-1.12.0-1.3"
				}
			},
			{
				"type": "Product Version",
				"name": "python312-rope-1.12.0-1.3",
				"full_product_name": {
					"text": "python312-rope-1.12.0-1.3",
					"product_id": "python312-rope-1.12.0-1.3"
				}
			}
		],
		"relationship": [
			{
				"product_reference": "python310-rope-1.12.0-1.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "python311-rope-1.12.0-1.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			},
			{
				"product_reference": "python312-rope-1.12.0-1.3",
				"relation_type": "Default Component Of",
				"relates_to_product_reference": "openSUSE Tumbleweed"
			}
		]
	},
	"vulnerability": [
		{
			"notes": [
				{
					"text": "base/oi/doa.py in the Rope library in CPython (aka Python) allows remote attackers to execute arbitrary code by leveraging an unsafe call to pickle.load.",
					"title": "Vulnerability Description",
					"type": "General"
				}
			],
			"cve": "CVE-2014-3539",
			"product_statuses": [
				{
					"type": "Fixed",
					"product_id": [
						"openSUSE Tumbleweed:python310-rope-1.12.0-1.3",
						"openSUSE Tumbleweed:python311-rope-1.12.0-1.3",
						"openSUSE Tumbleweed:python312-rope-1.12.0-1.3"
					]
				}
			],
			"threats": [
				{
					"type": "Impact",
					"description": "moderate"
				}
			],
			"cvss_score_sets": {
				"score_set_v_2": {
					"base_score_v_2": "7.5",
					"vector_v_2": "AV:N/AC:L/Au:N/C:P/I:P/A:P"
				},
				"score_set_v_3": {
					"base_score_v_3": "9.8",
					"vector_v_3": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:H/A:H"
				},
				"scoreset": {}
			},
			"remediations": [
				{
					"type": "Vendor Fix",
					"description": {
						"text": "To install this SUSE Security Update use the SUSE recommended installation methods like YaST online_update or \"zypper patch\".\n",
						"lang": "en"
					}
				}
			],
			"references": [
				{
					"url": "https://www.suse.com/security/cve/CVE-2014-3539.html",
					"description": "CVE-2014-3539"
				},
				{
					"url": "https://bugzilla.suse.com/916890",
					"description": "SUSE Bug 916890"
				}
			]
		}
	]
}
